connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -191,10 +191,66 @@ private class UCProxy extends TableCatalog with SupportsNamespaces {
       .asInstanceOf[Table]
   }
 
-  override def createTable(ident: Identifier, columns: Array[Column], partitions: Array[Transform], properties: util.Map[String, String]): Table = ???
+  override def createTable(ident: Identifier, columns: Array[Column], partitions: Array[Transform], properties: util.Map[String, String]): Table = {
```

Comment:
we can simply remove this method here, as the interface provides a default implementation.